### DISCUSSION TITLE
Idle Drop for "Banana" game

### DISCUSSION THREAD

**Discussion by Nazar1ky** [chr233/ASFEnhance#268] [功能建议 / Ideas]
I suggest to add some functional for Idle drop Banana game, maybe as separate plugin

**Comment by chr233** [comment]
seems there will be a lot of work to do

**Reply by Nazar1ky** [reply]
okay, thanks for answer!

**Reply by givrzeczy** [reply]
> seems there will be a lot of work to do

This would be wonderful if you could do it.

**Reply by chr233** [reply]
> > seems there will be a lot of work to do
> 
> This would be wonderful if you could do it.

I don't know very about how to do this
